packages/embark-ui/src/components/Communication.js

```diff
@@ -32,8 +33,10 @@ class Communication extends Component {
       [name]: e.target.value
     });
 
-    if(name === "channel") {
+    if (name === "channel") {
       this.setState({channelIsValid: e.target.value.length >= 4});
+    } else if (name === "listenTo") {
+      this.setState({listenToChannelIsValid: e.target.value.length >= 4});
```

Comment:
Tiny nitpick, maybe have a method that runs the validation instead of duplicating the logic would be best.

Reply:
Absolutely, will update right away.